charts/k8s-service/values.yaml

```diff
@@ -404,6 +404,25 @@ imagePullSecrets: []
 serviceAccount:
   name: ""
 
+# horizontalPodAutoscaler is a map that configures the Horizontal Pod Autoscaler information for this pod
+# The expected keys of hpa are:
+#   - enabled                      (bool)   : Whether or not Horizontal Pod Autoscaler should be created, if false the
+#                                             Horizontal Pod Autoscaler will not be created
+#   - minReplicas                  (int)    : The minimum amount of replicas allowed
+#   - maxReplicas                  (int)    : The maximum amount of replicas allowed
+#   - avgCpuUtilization            (int)    : The target average CPU utilization to be used with the metrics
+#   - avgMemoryUtilization         (int)    : The target average Memory utilization to be used with the metrics
+#
+# The default config will not create the Horizontal Pod Autoscaler by setting enabled = false, the default values are
+# set so if enabled is true the horizontalPodAutoscaler has valid values.
+horizontalPodAutoscaler:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 10
+  avgCpuUtilization: 50
+  avgMemoryUtilization: 50
```

Comment:
If we agree that we should have the if condition to allow disabling one of the conditions, I would recommend omitting `avgCpuUtilization` and `avgMemoryUtilization` from the default config so that you can opt in.